Makefile

```diff
@@ -8,6 +8,7 @@ check: check-licence check-go
 check-licence:
 	@(fgrep -rl "Licensed under the LGPLv3" .;\
 		fgrep -rl "MACHINE GENERATED BY THE COMMAND ABOVE; DO NOT EDIT" .;\
+		fgrep -rl "license that can be found in the LICENSE.ricochet2200 file" .; \
```

Comment:
Isn't this a regex? Does the '.' after LICENSE need to be escaped?

Reply:
fgrep = grep -F
"Interpret PATTERNS as fixed strings, not regular expressions."

Reply:
fgrep is also deprecated, changing all to 'grep -F'